src/com/sap/piper/JenkinsJobController.groovy

```diff
@@ -0,0 +1,97 @@
+package com.sap.piper
+
+class JenkinsJobController implements Serializable {
+    def script
+    def timeout = 3600
+    String jenkinsUrl = "http://localhost:8080"
```

Comment:
This is just a default value, groovy has implicit constructors which can (and should) be used to override this value, cf https://github.com/SAP/devops-docker-images/pull/21/files#diff-c9fd2b141b0b7791c38257322348e9e3R24

Reply:
> This is just a default value [...] which can **(and should)**

In case is should be overwritten I vote for not providing a default in order to make clear to other developers that is should be overwritten. Having "just default values" for the sake of having defaults blurs that it should be overwritten.

It is a question about awareness, like often in this business ... Awareness is higher in case there is no default in case there is no reasonable default. Other example where a default makes clearly sense to me: `timeout = 3600`.